ds_judgements_public_ui/templates/includes/results_applied_filters.html

```diff
@@ -1,38 +1,17 @@
-{# {% if form.search_term.data or (form.collections.data != 'all') or (form.courts.data != 'all') %} #}
-  <h2 class="results-search-component__sub-header">Filters include</h2>
-  <ul class="results-search-component__removable-options">
-    {# {% if form.search_term.data %} #}
+{% load query_filters %}
+<h2 class="results-search-component__sub-header">Filters include</h2>
+<ul class="results-search-component__removable-options">
+  {% for key, value in context.query_params.items %}
+    {% if value %}
       <li>
-        <span class="results-search-component__removable-options-key">Search term:</span>
+        <span class="results-search-component__removable-options-key">{{ key }}:</span>
```

Comment:
HI @LornaHarwood - I wonder if the key should be capitalised (possibly using [capfirst](https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#capfirst)